flocx_market/cmd/dbsync.py

```diff
@@ -6,4 +6,4 @@
 
 def main():
     flocx_market_service.prepare_service(sys.argv)
-    db_api.setup_db()
+db_api.setup_db()
```

Comment:
You probably don't want this change

Reply:
That was a mistake from copy-pasting too quick